main/meleeMenu.c

```diff
@@ -38,7 +38,7 @@ static const led_t borderLedColors[NUM_ROW_COLORS_AND_OFFSETS] =
     {.r = 0x10, .g = 0x10, .b = 0x20},
     {.r = 0x20, .g = 0x10, .b = 0x10},
     {.r = 0x00, .g = 0x20, .b = 0x10},
-    {.r = 0x20, .g = 0x20, .b = 0x00},
+    {.r = 0x20, .g = 0x20, .b = 0x10},
```

Comment:
This color shouldn't have changed. Now that menu changes were merged into `main`, can you rebase this branch off `main` so that menu changes aren't being reviewed?

Reply:
How would I go about doing that without either solving merge conflicts every commit or cherry-picking commits?